2023-04-26 13:02:16 [31m[1mERROR[0m HTTPSConnectionPool(host='noominatim.openstreetmap.org', port=443): Max retries exceeded with url: /search?street=east+4th+street+1&city=Ainsworth&county=+&state=+&country=+&postalcode=+&format=json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002211F5A62E8>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))